PocketCastsTests/Tests/Playback/Effects/AudioUtilsTests.swift

```diff
@@ -0,0 +1,172 @@
+import XCTest
+@testable import podcasts
+import AVFoundation
+import Accelerate
+
+final class AudioUtilsTests: XCTestCase {
+
+    var audioBuffer: AudioBuffer!
+
+    override func setUp() async throws {
+        audioBuffer = sampleBuffer()
+    }
+
+    override func tearDown() async throws {
+        cleanUpAudioBuffer(buffer: &audioBuffer)
+    }
+
+    private var bufferFormat: AVAudioFormat = {
+        var asbd = AudioStreamBasicDescription(
+            mSampleRate: 44100.0,
+            mFormatID: kAudioFormatLinearPCM,
+            mFormatFlags: kAudioFormatFlagIsFloat | kAudioFormatFlagIsPacked,
+            mBytesPerPacket: UInt32(MemoryLayout<Float32>.size),
+            mFramesPerPacket: 1,
+            mBytesPerFrame: UInt32(MemoryLayout<Float32>.size),
+            mChannelsPerFrame: 1,
+            mBitsPerChannel: UInt32(8 * MemoryLayout<Float32>.size),
+            mReserved: 0
+        )
+
+        return AVAudioFormat(streamDescription: &asbd)!
+    }()
+
+    private func sampleBuffer() -> AudioBuffer {
+        let samples: [Float32] = [-8, -4, -0, 2, 4, 8]
+
+        var audioBuffer = AudioBuffer()
+        audioBuffer.mNumberChannels = 1 // Mono audio
```

Comment:
When using `EffectsPlayer` we always convert mono to stereo, as per: https://github.com/Automattic/pocket-casts-ios/pull/213

Shall we update the tests here?

Reply:
It doesn't seem to make a difference to the test results. I'll set it to 2 channels just to make it better match the handling from within `EffectsPlayer`.